app/models/audio_version.rb

```diff
@@ -9,7 +9,7 @@ class AudioVersion < BaseModel
 
   def length(reload=false)
     @_length = nil if reload
-    @_length ||= self.audio_files.inject(0){|sum, f| sum + f.length}
+    @_length ||= self.audio_files.inject(0){|sum, f| sum + f.length.to_i}
```

Comment:
Redundant `self` detected.<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.